src/mds/MDCache.h

```diff
@@ -157,10 +157,15 @@ class MDCache {
   bool exceeded_size_limit;
 
 private:
+  MemoryModel mm(g_ceph_context);
+  MemoryModel::snap last;
```

Comment:
`last` should still be a stack variable in `MDCache::resize`.

Reply:
Ack updated it